app/Event/EventGeoCoordinatesServiceProvider.php

```diff
@@ -21,12 +21,16 @@ public function register(Application $app)
     {
         $app['event_geocoordinates_command_handler'] = $app->share(
             function (Application $app) {
-                return new GeoCoordinatesCommandHandler(
+                $handler = new GeoCoordinatesCommandHandler(
                     $app['event_repository'],
                     new DefaultAddressFormatter(),
                     new LocalityAddressFormatter(),
                     $app['geocoding_service']
                 );
+
+                $handler->setLogger($app['logger.command_bus']);
```

Comment:
We included it in this log because the `DefaultGeoCodingService` also uses this logger.